atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTableManipulationTest.java

```diff
@@ -17,9 +17,49 @@
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
 
 public class OneNodeDownTableManipulationTest {
+    private static final TableReference NEW_TABLE = TableReference.createWithEmptyNamespace("new_table");
+    private static final TableReference NEW_TABLE2 = TableReference.createWithEmptyNamespace("new_table2");
+
+    @Rule
+    public ExpectedException expectException = ExpectedException.none();
+
+    @Test
+    public void createTableThrows() {
+        expectException.expect(IllegalStateException.class);
```

Comment:
I suggest we either always use the `expectException` or the `assertThatThrownBy` route in all the tests in this class, for consistency. I don't actually mind either way which approach! The assertj way would be:

```
assertThatThrownBy(() -> OneNodeDownTestSuite.db.dropTable(OneNodeDownTestSuite.TEST_TABLE))
    .isInstanceOf(IllegalStateException.class)
    .hasMessage("Creating tables requires all Cassandra nodes to be up and available.");
```